src/test/common/clara/test_bindings.cljc

```diff
@@ -369,3 +371,70 @@
              fire-rules
              (query cold-query)))
       "The query results should not be empty for a matching location"))
+
+;; https://github.com/cerner/clara-rules/issues/426
+;; A join binding used in the same condition but different constraint will fail to compile.
+(defn calc-wind-chill
+  [temp wind-speed]
+  (+ 35.74
+     (- (* 0.6215 temp)
+        (* 35.75
+           (Math/pow wind-speed 0.16)))
+     (* 0.4275 temp (Math/pow wind-speed 0.16))))
```

Comment:
I'm guessing this is a formula for wind chill?  It seems unnecessarily complicated to use a real formula here since what we're testing has nothing to with this.  Also does "Math/pow" work in ClojureScript?